server/model/traqMessage.go

```diff
@@ -0,0 +1,97 @@
+package model
+
+import "strings"
+
+func TraqMessageProcessor(messageList MessageList) (SendList, error) {
+	words := []Words{}
+	err := db.Select(&words, "SELECT words.word, words.bot_notification, words.me_notification, words.trap_id, users.traq_uuid, users.is_bot, users.dm_id FROM words INNER JOIN users ON users.trap_id = words.trap_id")
```

Comment:
`を追加すると改行が使えるようになるので、こういった長いSQLクエリなどは見やすくなるよう整形すると良いです
```suggestion
	err := db.Select(&words, `
	SELECT words.word, words.bot_notification, words.me_notification, words.trap_id, users.traq_uuid, users.is_bot, users.dm_id
	FROM words 
	INNER JOIN users ON users.trap_id = words.trap_id`)
```

Reply:
ここでJoinせずに、wordsとusersの情報を別々で持ってたほうがいいのでは
usersのデータはmapに変換して引きやすくするといいかも
そうすればbotかどうかの判定でDBに都度問い合わせる必要なくなります